### PR TITLE
feat(route): add BT之家1LOU站

### PR DESCRIPTION
**Pull request by falling**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/1lou/
/1lou/search-繁花.htm
/1lou/forum-1.htm
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
- [ ] Full text / 全文获取
  - [x] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明
1LOU 论坛的RSS

### CODE REVIEW

File: website/docs/routes/multimedia.mdx [hunk 1]
```diff
@@ -1587,14 +1587,20 @@ Refer to [https://developers.themoviedb.org/3/getting-started/languages](https:/
...
 
-## 综艺秀 {#zong-yi-xiu}
+## BT之家 1LOU站 {#bt-zhi-ji-1Lou}
 
-### 综艺 {#zong-yi-xiu-zong-yi}
+### 搜索 {#bt-zhi-ji-1Lou-sou-suo}
 
-<Route author="pharaoh2012 nczitzk" example="/zyshow/chongchongchong" path="/zyshow/:region?/:id" paramsDesc={['地区，见下表，默认为空，即台湾', '综艺 id，综艺详情对应页 URL 中找到']} radar="1" anticrawler="1">
-  地区
+<Route author="falling" example="/1lou/search-_E5_8B_BF_E8_A8_80_E6_8E_A8_E7_90_86" path="/1lou/:path" paramsDesc={['路径信息在URL里找到,主页为 index']}>
```
Comment (TonyRL) [inline website/docs/routes/multimedia.mdx:1594]: Do not delete other route's docs

File: lib/v2/1lou/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        items.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const detailResponse = await got({
+                    method: 'get',
+                    url: item.link,
+                });
+                const content = cheerio.load(detailResponse.data);
+                const torrents = content('.attachlist').find('a');
+                item.description = content('.message.break-all').html();
+                if (torrents.length > 0) {
```
Comment (TonyRL) [inline lib/v2/1lou/index.js:39]:
```suggestion
                item.description = content('.message.break-all').html();
                if (torrents.length > 0) {
                const torrents = content('.attachlist').find('a');
```

File: lib/v2/1lou/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const path = ctx.params.path ?? '';
+    const rootUrl = `https://www.1lou.me`;
+    const currentUrl = `${rootUrl}/${path}.htm`;
```
Comment (TonyRL) [inline lib/v2/1lou/index.js:9]: If `path` is empty, RSSHub will be visiting `https://www.1lou.me/.htm`

File: lib/v2/1lou/index.js (hunk 2 above)
Comment (falling) [reply lib/v2/1lou/index.js:39]: we can't use **torrents.length** before the const
```js
if (torrents.length > 0) {
    const torrents = content('.attachlist').find('a');
```

File: lib/v2/1lou/radar.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    '1lou.me': {
+        _name: 'BT之家 1LOU站',
+        '.': [
+            {
+                title: '搜索',
+                docs: 'https://docs.rsshub.app/routes/multimedia#bt-zhi-ji-1Lou',
```
Comment (TonyRL) [inline lib/v2/1lou/radar.js:7]:
```suggestion
                docs: 'https://docs.rsshub.app/routes/multimedia#bt-zhi-jia-1lou-zhan',
```

File: website/docs/routes/multimedia.mdx [hunk 5]
```diff
@@ -1598,3 +1598,21 @@ Refer to [https://developers.themoviedb.org/3/getting-started/languages](https:/
   | ---- | ---- | ---- |
   |      | kr   | dl   |
 </Route>
+
+## BT之家 1LOU站 {#bt-zhi-ji-1Lou}
```
Comment (TonyRL) [inline website/docs/routes/multimedia.mdx:1602]:
```suggestion
## BT之家 1LOU站 {#bt-zhi-jia-1lou-zhan}
```

File: website/docs/routes/multimedia.mdx [hunk 6]
```diff
@@ -1598,3 +1598,21 @@ Refer to [https://developers.themoviedb.org/3/getting-started/languages](https:/
...
+
+### 搜索 {#bt-zhi-ji-1Lou-sou-suo}
+
+<Route author="falling" example="/1lou/search-繁花.htm" path="/1lou/:path" paramsDesc={['路径信息在URL里找到,主页为 index']}>
```
Comment (TonyRL) [inline website/docs/routes/multimedia.mdx:1606]: https://github.com/DIYgod/RSSHub/blob/c29cf67664fe40a20ffb636ed90b2cc20facdc62/lib/v2/1lou/maintainer.js#L2
```suggestion
<Route author="falling" example="/1lou/search-繁花.htm" path="/1lou/:path?" paramsDesc={['路径信息在URL里找到,主页为 index']}>
```

**Pull request merged**